man/system_derivatives.Rd

```diff
@@ -83,9 +83,7 @@ soybean_system <- system_derivatives(
 
 derivs <- soybean_system(0, unlist(soybean$initial_values), NULL)
 
-# Example 2: a simple oscillator with only one module (requires deSolve)
```

Comment:
Maybe you should have just changed this to "requires patience".  I'm being a little facetious, but still, although you warn the user that Example 3 will run very slowly, it's probably worth putting a very prominent warning right at the top of the examples section that will appear next to the "Run examples" link.  Otherwise users may begin to wonder, as I did, whether the "Run examples" link really does anything.

Also note that removing "dontrun" will cause _pkgdown_ to take considerably longer to build the reference section of the pkgdown site.  I don't see this as a real problem—the pkgdown documentation really need only be built once for each new release, and it's probably a _good_ thing that people who go to the public site to view the documentation don't have to wait around to see the result of running the example.  But it's something to be aware of.

Reply:
The warning is a good idea. I think the extra time for building the pkgdown documentation is acceptable.